Projekty/Projekt1/AdriannaGrudzień_KingaUłasik/EDA_grades.ipynb

```diff
@@ -0,0 +1,2485 @@
+{
```

Comment:
<p>To, że coś jest kropką na boxplocie, to nie znaczy, że jest outlierem.</p><p> </p><p>Trochę naćkane tych wykresów. Nie pamiętam, czy pisałem o tym ostatnio, ale na pewno myślałem to samo \:)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>